dbms/src/Interpreters/Settings.h

```diff
@@ -75,6 +75,7 @@ struct Settings
     \
     M(SettingBool, compile, false, "Whether query compilation is enabled.") \
     M(SettingBool, compile_expressions, false, "Compile some scalar functions and operators to native code.") \
+    M(SettingUInt64, compiled_expressions_cache_size, std::numeric_limits<UInt64>::max(), "Cache size for compiled expressions") \
```

Comment:
The place for this setting is non consistent with the corresponding settings for MarkCache and UncompressedCache.

Reply:
Both are server level settings (config.xml).